special-interest-groups/sig-dashboard/roles-and-organization-management.md

```diff
@@ -0,0 +1,75 @@
+# Dashboard SIG Roles and Organization Management
+
+## Join and Promotion
+
+We mainly follow the instructions in [TiDB Developer Group](../../architecture/README.md#tidb-developer-group).
```

Comment:
Maybe /architecture/README.md#tidb-developer-group is enough?

Reply:
In GitHub preview this kind of link is broken.. not sure whether it will work after merged.